.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,47 @@
+name: pre-commit
+
+on:
+  push:
+    branches:
+      - develop
+      - master
```

Comment:
`master` has been retired, we can go without this now

Reply:
good catch yeah